### PR TITLE
Preloader page added under the name src

### PR DESCRIPTION
**Pull request by deekshitLD**
Please Check and Tell

### CODE REVIEW

File: src/project1.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>My Webpage new</title>
+    <link rel="stylesheet" href="C:/Users/Deeksha/Desktop/project/project1.css">
```
Comment (ricksondpenha) [inline src/project1.html]: never put paths that link to your system path, always make sure all paths are in project level only.

**Review by ricksondpenha — CHANGES_REQUESTED** [review]
fix the path

File: src/project1.html (hunk 1 above)
Comment (ricksondpenha) [reply src/project1.html]: you can always find relative path by right click on the css / js file and get relative path option

Comment (deekshitLD) [reply src/project1.html]: Yes understood.

**Review by ricksondpenha — APPROVED** [review]

**Pull request merged**